frontend/src/metabase/admin/databases/components/CreatedDatabaseModal.jsx

```diff
@@ -1,44 +1,32 @@
+/* @flow */
 import React, { Component } from "react";
-import PropTypes from "prop-types";
 import { Link } from "react-router";
-import { t, jt } from "c-3po";
-import ModalContent from "metabase/components/ModalContent.jsx";
+import { t } from "c-3po";
 
-import * as Urls from "metabase/lib/urls";
+import ModalContent from "metabase/components/ModalContent.jsx";
 
+type Props = {
+  onClose: () => void,
+  onDone: () => void,
+};
 export default class CreatedDatabaseModal extends Component {
-  static propTypes = {
-    databaseId: PropTypes.number.isRequired,
-    onClose: PropTypes.func.isRequired,
-    onDone: PropTypes.func.isRequired,
-  };
+  props: Props;
 
   render() {
-    const { onClose, onDone, databaseId } = this.props;
```

Comment:
we should hold onto the databaseId and change the `/ready` url to a `ready/databaseId` url so that we can have a re-entrant "here's the auto dashboards for this database" landing page